### ISSUE TITLE
Dashboard

### ISSUE THREAD

**Issue by NJaku01** [o2r-project/o2r-UI#89]
@nuest a dashboard is to be created for the UI. Here information about ERCs shall be displayed. This information should be interesting for publishers and authors. Enclosed is the first draft of a dashboard as mock-up. Do you have any ideas, criticism or suggestions for improvement? 
![dashboard_mockup](https://user-images.githubusercontent.com/46575458/72151308-29fa1900-33a8-11ea-9337-ffcd5ecd4671.png)

**Comment by nuest** [comment]
@NJaku01 Sorry, this slipped by. Is the mock-up still current? Please ping me on Mattermost if I'm still allowed to give feedback :-)

**Comment by nuest** [comment]
Good idea. It will be useful to track this, and having an idea about the numbers we need to collect will help us a lot in deciding _how_ to collect them.

For now I have mostly questions:

- Could you show some raw numbers on the "most inspected ERCs"? Maybe make this paginated list so I can also see beyond the top 5.
- What do you count as an "inspection"? One execution of a binding?
- "Reproducibility analysis" > rename to "Check statistics". The system does not know if something is really reproducible, it can just tell us if the execution failed or not.
- What is the map showing? Bounding boxes from the ERC metadata would make most sense to me..
- Which user level would you allow this to see? Or may anyone access it?
- I suggest to reconsider all headlines to match terms/words from the regular UI. Why is there no "Manipulation statistics" ?
- Suggest to split "ERC uploads" into workspace uploads, ERC uploads, cloud-based uploads (e.g. with stacked bar chart)
- What about user statistics (user with most ERCs, number of registered users) ?

Maybe that's enough for the current iteration :-) Let me know if something doesn't make sense to you, happy to take a look at the next draft again!

**Comment by NJaku01** [comment]
The statistical data should be added to the metadata of the ERCs. How exactly the data will be collected is not clear yet.

- As raw data I now have the number of visits to the ERC and if the last check analysis was successful.

- It was planned that even entering the ERC site counts as an "inspection".

- Here now also the number of analyses are displayed. Therefore I decided to rename the graph to check analyses

- Right, the map should refer to the bounding boxes. The idea here was not to show the bounding boxes as polygons but to show the center of the bounding boxes to keep the map clear. The center points should be clustered.

- The dashboard should be accessible for everyone.

- In the Manipulation statistics, the number of times the corresponding window has been opened is now shown. This has now been set in comparison to Inspect, Check, and Substitution.

- This will be implemented as soon as the cloud-based upload is possible.

- User statistics should not be shown for the moment after arrangement with Markus.

Enclosed is the new version of the dashboard mock-up, but the dashboard will be moved back a bit because I have another task to do first.
![dashboard_mockup(1)(2)](https://user-images.githubusercontent.com/46575458/74934930-932d6d00-53e7-11ea-960c-970b89bb9be9.png)

**Comment by nuest** [comment]
Thanks for the update!

Note that since there is already an Elasticsearch server running, IMO it would be wise (and I would recommend against developing a standalone dashboard solution) to take a very good look at Kibana: https://www.elastic.co/kibana

**Comment by nuest** [comment]
Note that currently we have disabled Elasticsearch, because we don't use the search features. I think for the number of ERCs we're realistically expecting, the dashboard could also be realised with the MongoDB. And maybe with Shiny?

**Comment by NJaku01** [comment]
https://github.com/o2r-project/o2r-UI/tree/dashboard
